lib/components/SelectionControl/radio/radio-button/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedCircleMdRegularIcon,
+    LabelPairedCircleDotMdFillIcon,
+    LabelPairedCircleInfoMdRegularIcon,
+} from "@deriv/quill-icons";
+import { Text } from "../../../Typography";
+import "./radio-button.scss";
+
+type TRadio = {
+    className?: string;
+    classNameInfo?: string;
+    classNameLabel?: string;
+    defaultChecked?: boolean;
+    disabled?: boolean;
+    has_info?: boolean;
+    id?: string;
+    name?: string;
+    onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    required?: boolean;
+    value?: string;
+};
+
+const RadioButton = ({
+    children,
+    className,
+    classNameInfo,
+    classNameLabel,
+    defaultChecked,
+    disabled = false,
+    has_info,
```

Comment:
Should it accept size? In Figma it can be sm and md

Reply:
Hmm, I'll check. I didn't see small size in figma. I'll look closer and check.